actions/terraform/plan-only/action.yaml

```diff
@@ -0,0 +1,247 @@
+name: Terraform Plan
```

Comment:
This was copy pasted from the other action and I got confused why it said Terraform Plan instead of Terraform Apply.

Reply:
I meant to write this in here: https://github.com/Altinn/altinn-platform/pull/1540/files#r2039182074